plugins/catalog-backend/package.json

```diff
@@ -73,7 +73,7 @@
     "@types/yup": "^0.29.13",
     "aws-sdk-mock": "^5.2.1",
     "msw": "^0.35.0",
-    "sqlite3": "^5.0.1",
+    "sqlite3": "git://github.com/mapbox/node-sqlite3.git#918052b538b0effe6c4a44c74a16b2749c08a0d2",
```

Comment:
Does sqlite3 work well with duplicate installs? Most ppl would end up with that due to this afaik